app/scripts/modules/openstack/serverGroup/configure/wizard/advanced/advancedSettings.component.html

```diff
@@ -11,18 +17,25 @@
     </div>
   </div>
   <div class="form-group">
-    <div class="col-md-3 sm-label-right">User Data</div>
-    <div class="col-md-7 no-spel" ng-switch on="$ctrl.command.userDataType">
+    <div class="col-md-3 sm-label-right">User Data Value</div>
+    <div class="col-md-9 no-spel" ng-switch on="$ctrl.command.userDataType">
```

Comment:
the `ng-switch` isn't doing anything (I know it was already there, just hadn't noticed it before)